### PR TITLE
docs: remove duplicated contents

### PR DESCRIPTION
**Pull request by noyidoit**
去除了原文

### CODE REVIEW

File: docs/zh/guide/essentials/nested-routes.md [hunk 1]
```diff
@@ -94,4 +92,4 @@ const routes = [
 ]
 ```
 
-这个例子的 demo 可以在[这里](https://codesandbox.io/s/nested-views-vue-router-4-examples-hl326?initialpath=%2Fusers%2Feduardo)找到。
\ No newline at end of file
+这个例子的 demo 可以在[这里](https://codesandbox.io/s/nested-views-vue-router-4-examples-hl326?initialpath=%2Fusers%2Feduardo)找到。
```
Comment (posva) [inline docs/zh/guide/essentials/nested-routes.md:95]: What is the change in this line?

Comment (noyidoit) [reply docs/zh/guide/essentials/nested-routes.md:95]: I didn't change anything in this line, I guess it's a flaw of Github's diff.

Comment (posva) [inline docs/zh/guide/essentials/nested-routes.md:95]: Weird, let's test something

```suggestion
这个例子的 demo 可以在[这里](https://codesandbox.io/s/nested-views-vue-router-4-examples-hl326?initialpath=%2Fusers%2Feduardo)找到。
```

**Comment by posva** [comment]
Thanks! I ended up applying the diff locally just in case

**Pull request closed without merging**